lib/restclient/payload.rb

```diff
@@ -116,9 +116,12 @@ def inspect
       end
 
       def short_inspect
-        (size > 500 ? "#{size} byte(s) length" : inspect)
+        if RestClient.limit_payload_log? && size > 500
+          "#{size} byte(s) length"
+        else
+          inspect
```

Comment:
We probably want to warn people not to run this in production and it could possibly leak sensitive information into logs.